docs/modules/druid/pages/usage-guide/operations/graceful-shutdown.adoc

```diff
@@ -1,7 +1,179 @@
+druid 2023-11-13T10:56:54,089 INFO [Thread-55] org.eclipse.jetty.server.session - node0 Stopped scavenging
+druid 2023-11-13T10:56:54,090 INFO [Thread-55] org.eclipse.jetty.server.handler.ContextHandler - Stopped o.e.j.s.ServletContextHandler@3b9d85c2{/,null,STOPPED}
+druid 2023-11-13T10:56:54,093 INFO [Thread-55] org.apache.druid.java.util.common.lifecycle.Lifecycle - Stopping lifecycle [module] stage [NORMAL]
+druid 2023-11-13T10:56:54,094 INFO [Thread-55] org.apache.druid.server.coordination.ZkCoordinator - Stopping ZkCoordinator for [DruidServerMetadata{name='druid-historical-default-0.druid-historical-default.kuttl-test-striking-polliwog.svc.cluster.local:8283', hostAndPort='null', hostAndTlsPort='druid-historical-default-0.druid-historical-default.kuttl-test-striking-polliwog.svc.cluster.local:8283', maxSize=1000000000, tier='_default_tier', type=historical, priority=0}]
+druid 2023-11-13T10:56:54,094 INFO [Thread-55] org.apache.druid.server.coordination.SegmentLoadDropHandler - Stopping...
+druid 2023-11-13T10:56:54,094 INFO [Thread-55] org.apache.druid.server.coordination.SegmentLoadDropHandler - Stopped.
+druid 2023-11-13T10:56:54,094 INFO [LookupExtractorFactoryContainerProvider-MainThread] org.apache.druid.query.lookup.LookupReferencesManager - Lookup Management loop exited. Lookup notices are not handled anymore.
+druid 2023-11-13T10:56:54,096 INFO [Thread-55] org.apache.druid.security.basic.authorization.db.cache.CoordinatorPollingBasicAuthorizerCacheManager - CoordinatorPollingBasicAuthorizerCacheManager is stopping.
+druid 2023-11-13T10:56:54,096 INFO [Thread-55] org.apache.druid.security.basic.authorization.db.cache.CoordinatorPollingBasicAuthorizerCacheManager - CoordinatorPollingBasicAuthorizerCacheManager is stopped.
+druid 2023-11-13T10:56:54,096 INFO [Thread-55] org.apache.druid.security.basic.authentication.db.cache.CoordinatorPollingBasicAuthenticatorCacheManager - CoordinatorPollingBasicAuthenticatorCacheManager is stopping.
+druid 2023-11-13T10:56:54,096 INFO [Thread-55] org.apache.druid.security.basic.authentication.db.cache.CoordinatorPollingBasicAuthenticatorCacheManager - CoordinatorPollingBasicAuthenticatorCacheManager is stopped.
+druid 2023-11-13T10:56:54,108 INFO [Curator-Framework-0] org.apache.curator.framework.imps.CuratorFrameworkImpl - backgroundOperationsLoop exiting
+druid 2023-11-13T10:56:54,212 INFO [Thread-55] org.apache.zookeeper.ZooKeeper - Session: 0x10011760e0e0007 closed
+druid 2023-11-13T10:56:54,212 INFO [main-EventThread] org.apache.zookeeper.ClientCnxn - EventThread shut down for session: 0x10011760e0e0007
+druid 2023-11-13T10:56:54,212 INFO [Thread-55] org.apache.druid.java.util.common.lifecycle.Lifecycle - Stopping lifecycle [module] stage [INIT]
+----
+
+== MiddleManager
+
+As a default, Druid middle managers have `5 minutes` to shut down gracefully.
+
+They use the same mechanism described above.
```

Comment:
I'm confused. Below we say something different:

>              // The middle manager can be terminated gracefully by disabling it, meaning
>               // the overlord will not send any new tasks and it will terminate after
>               // all tasks are finished or the termination grace period is exceeded.

Reply:
replaced in docs